apps/next-evans/components/pages/friend/MapBlock.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { t } from '@friends-library/locale';
+import Image from 'next/image';
+import Uk from '../../../public/images/maps/UK--2x.png';
+import Us from '../../../public/images/maps/US.png';
+import Europe from '../../../public/images/maps/Europe--2x.png';
```

Comment:
is it possible to use the `@/<...>` alias for these? if so, change em all -- there are a few other files in this PR with them too